### ISSUE TITLE
create-react-class should be dependency instead of devDependency

### ISSUE THREAD

**Issue by bengummer** [martinandert/react-lorem-component#6]
The `create-react-class` class was introduced in 0.12.0, which is then used in `index.js` ([source](https://github.com/martinandert/react-lorem-component/commit/375d348f41c55d64994f2606b99527c02286c9b4#diff-168726dbe96b3ce427e7fedce31bb0bcR4)).

This causes issues when `react-lorem-component` is installed without devDependencies, because `create-react-class` cannot be found:

```
ERROR in ./packages/modal-dialog/~/react-lorem-component/index.js
Module not found: Error: Cannot resolve module 'create-react-class' in /Users/ben/Repos/atlassian/atlaskit/packages/modal-dialog/node_modules/react-lorem-component
 @ ./packages/modal-dialog/~/react-lorem-component/index.js 4:23-52
```

Moving the `create-react-class` line in `package.json` from `devDependencies` to `dependencies` should fix this.

**Comment by bengummer** [comment]
Let me know if a PR would be useful, happy to help although just a 1 line change 😄

**Comment by martinandert** [comment]
Hi @bengummer,

Thanks for reporting! It's fixed in v0.12.1.

**Comment by bengummer** [comment]
Thanks @martinandert :)
